awslabs/amazon-timestream-driver-jdbc
# [Issue-35] remove log configuration and add example for debug

## Summary
Our code has a static block that creates a sl4j bridge and set up a log level for the whole application.
That behavior overrides the default application configuration.
With the fix redundant code blocks has been removed and documentation on how to trace using the different application has been added to README.md

## Description
The driver overrides LOG level for the application. (Application print logs that are available only for trace/debug purposes)

## Related Issue
[Issue-35](https://github.com/awslabs/amazon-timestream-driver-jdbc/issues/35)

## Tests performed/created
### Unit tests: 
All existing Unit tests passed

### Manual verification:
1. Build the project with the command `mvn clean install`
2. Use the signed jar in `DBeaver`
3. Create a timestream connection
4. Open Logs and verify that timestream driver logs present

1. Build the project with the command `mvn clean install`
2. Use the signed jar in `new java project as a dependency
3. Create a timestream connection
4. Open Logs and verify that timestream driver logs present
5. Change the Log level to debug
6. Verify that additional driver logs are added to the log file

## Additional Reviewers
@alexey-temnikov
